index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Some of the items seem a little bit off. For instance, the navbar looks like it is thrown to the left. Maybe this has to do with the size of the screen. One piece of advice is that you use a max-width property on the body, so everything is with a specific size.

> ![image](https://user-images.githubusercontent.com/57414789/82129760-a5bb0680-978a-11ea-8d64-c98400ecd797.png)